test/MyNFT.test.js

```diff
@@ -0,0 +1,158 @@
+const MyNFT = artifacts.require("./MyNFT");
+
+const {
+  ether,           // Big Number support
+  constants,    // Common constants, like the zero address and largest integers
+  expectEvent,  // Assertions for emitted events
+  expectRevert,
+  balance,
+  time, // Assertions for transactions that should fail
+} = require('@openzeppelin/test-helpers');
+
+const { expect } = require("chai");
+const { Web3 } = require('hardhat');
+const BN = Web3.utils.BN;
+
+contract("MyNFT", async ([owner, acc1, acc2, acc3, acc4, acc5, acc6]) => {
+
+  let baseURI = "https://gateway.pinata.cloud/ipfs/QmcUH66aaS6iCLJdDCas845xkzcRYgq8NNwfs3FMir8wTs/";
+
+  before(async () => {
+    instanceNFT = await MyNFT.new(ether('0.001'), baseURI);
+  });
+
+  describe("Initialize contract - false", async () => {
+    it("Error : nft price can't be equal 0", async () => {
+      await expectRevert(MyNFT.new(ether('0'), baseURI), "Error : nft price can't be equal 0");
+    });
+    it("Error : not valid URI", async () => {
+      await expectRevert(MyNFT.new(ether('1'), ""), "Error : not valid URI");
+    });
+  });
+
+  describe("setNftPrice function", async () => {
+
+    describe("False", async () => {
```

Comment:
false it's not corectly name for describe with fail tests
rename to: "Should fail if..."